### ISSUE TITLE
master.zip and 3.1.6-08 installation throwing error / CentOS 7

### ISSUE THREAD

**Issue by mzico** [GluuFederation/cluster-mgr#267]
Trying to install with `pip install https://github.com/GluuFederation/cluster-mgr/archive/3.1.6-08.tar.gz` and `pip install https://github.com/GluuFederation/cluster-mgr/archive/master.zip` in AWS.  Installation went okay I guess but starting clustermgr-cli throwing below error: 

```
[root@ip-172-31-34-216 ~]# clustermgr-cli start
Upgrading Database
Traceback (most recent call last):
  File "/bin/clusterapp.py", line 22, in <module>
    app = create_app()
  File "/usr/lib/python2.7/site-packages/clustermgr/application.py", line 45, in create_app
    silent=True,
  File "/usr/lib/python2.7/site-packages/flask/config.py", line 132, in from_pyfile
    exec(compile(config_file.read(), filename, "exec"), d.__dict__)
  File "/root/.clustermgr/instance/config.py", line 1
    DEBUG = False\nSECRET_KEY = 'wxJgbQyaxFeWpu1Ek5uSztiHKIyDfznY'\nLICENSE_ENFORCEMENT_ENABLED = True
                                                                                                     ^
SyntaxError: unexpected character after line continuation character
Starting Celery Worker
celery multi v4.3.0 (rhubarb)
> Starting nodes...
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib/python2.7/site-packages/celery/__main__.py", line 20, in <module>
    main()
  File "/usr/lib/python2.7/site-packages/celery/__main__.py", line 16, in main
    _main()
  File "/usr/lib/python2.7/site-packages/celery/bin/celery.py", line 322, in main
    cmd.execute_from_commandline(argv)
  File "/usr/lib/python2.7/site-packages/celery/bin/celery.py", line 496, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 288, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 502, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 524, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/usr/lib/python2.7/site-packages/celery/app/utils.py", line 368, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 527, in symbol_by_name
    return imports.symbol_by_name(name, imp=imp)
  File "/usr/lib/python2.7/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/usr/lib/python2.7/site-packages/celery/utils/imports.py", line 111, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/bin/clusterapp.py", line 22, in <module>
    app = create_app()
  File "/usr/lib/python2.7/site-packages/clustermgr/application.py", line 45, in create_app
    silent=True,
  File "/usr/lib/python2.7/site-packages/flask/config.py", line 132, in from_pyfile
    exec(compile(config_file.read(), filename, "exec"), d.__dict__)
  File "/root/.clustermgr/instance/config.py", line 1
    DEBUG = False\nSECRET_KEY = 'wxJgbQyaxFeWpu1Ek5uSztiHKIyDfznY'\nLICENSE_ENFORCEMENT_ENABLED = True
                                                                                                     ^
SyntaxError: unexpected character after line continuation character
        > worker@ip-172-31-34-216.us-west-2.compute.internal: * Child terminated with exit code 1
FAILED
Starting Celery Beat
Traceback (most recent call last):
  File "/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/celery/__main__.py", line 16, in main
    _main()
  File "/usr/lib/python2.7/site-packages/celery/bin/celery.py", line 322, in main
    cmd.execute_from_commandline(argv)
  File "/usr/lib/python2.7/site-packages/celery/bin/celery.py", line 496, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 288, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 502, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 524, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/usr/lib/python2.7/site-packages/celery/app/utils.py", line 368, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/usr/lib/python2.7/site-packages/celery/bin/base.py", line 527, in symbol_by_name
    return imports.symbol_by_name(name, imp=imp)
  File "/usr/lib/python2.7/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/usr/lib/python2.7/site-packages/celery/utils/imports.py", line 111, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/bin/clusterapp.py", line 22, in <module>
    app = create_app()
  File "/usr/lib/python2.7/site-packages/clustermgr/application.py", line 45, in create_app
    silent=True,
  File "/usr/lib/python2.7/site-packages/flask/config.py", line 132, in from_pyfile
    exec(compile(config_file.read(), filename, "exec"), d.__dict__)
  File "/root/.clustermgr/instance/config.py", line 1
    DEBUG = False\nSECRET_KEY = 'wxJgbQyaxFeWpu1Ek5uSztiHKIyDfznY'\nLICENSE_ENFORCEMENT_ENABLED = True
                                                                                                     ^
SyntaxError: unexpected character after line continuation character
Starting Gunicorn Web Server
```

**Comment by mzico** [comment]
I think this is basically a problem with uninstallation. 
When I move to uninstall/install from pip; it's happening.

**Comment by mbaser** [comment]
closed by commit 2e8c63b3254d7c09e719699d8d05f2b0b29829e3
